worlds/Arena.wbt

```diff
@@ -1,7 +1,7 @@
 #VRML_SIM R2020b utf8
 WorldInfo {
   coordinateSystem "NUE"
-  basicTimeStep 16
+  basicTimeStep 8
```

Comment:
👍 from my experiments earlier this didn't seem to change robot behavior visibly, may need to warn teams that it's happening though

Reply:
When we publish updates we should also publish a changelog anyway in GitHub releases, so that covers that I think.